sparkplug-extras/src/test/scala/springnz/sparkplug/orientdb/OrientRDDFetcherTest.scala

```diff
@@ -44,8 +47,23 @@ class OrientRDDFetcherTest extends WordSpec with ShouldMatchers with ODBMemoryTe
     resultWithWhereClause shouldBe (1 to numDocs / 2).toList
   }
 
+  "Save SparkRDD to OrientDB" in new SimpleTestContext("") {
+    val dictators = List(Dictator(1971, "Idi", "Uganda"), Dictator(1980, "Robert", "Zimbabwe"), Dictator(2009, "Jacob", "South Africa"))
+    val operation: SparkOperation[Set[Dictator]] = SparkOperation { ctx ⇒
+      val rdd = ctx.parallelize(dictators)
+      rdd.saveToOrient("Dictators")
+      val retrievedRDD = OrientRDDFetcher.select[Dictator]("Dictators", Some("year > 1979"), doc ⇒ {
+        Dictator(doc.field("year").asInstanceOf[Int], doc.field("name").asInstanceOf[String], doc.field("country").asInstanceOf[String])
```

Comment:
No problem. But FYI, if you import `ODBScala._` from the `orientdb-migrations` project, it gives you a `DocPimper` with `.getInt`, `.getString`, etc.